backend/commitments/views.py

```diff
@@ -7,7 +7,31 @@
 
 # Create your views here.
 
-# read one commitment
+
+# read all commitment for a specific client
+@require_GET
+def get_all_client(request, id_client):
```

Comment:
id client is in "authentificated_user.id" -> pas besoinde le mettre dans le /... de la route